src/apis/impl/AuthenticationApiImpl.ts

```diff
@@ -135,19 +152,19 @@ export class AuthenticationApiImpl implements AuthenticationApi {
           delegate
         );
         //TODO Return which delegates were warned to share back info & add retry mechanism
-        if (!accessNotification) throw new Error(`Cannot create a notification to Healthcare Professional with id ${delegate}`)
+        if (!accessNotification) throw this.errorHandler.createErrorWithMessage(`Cannot create a notification to Healthcare Professional with id ${delegate}`)
       }
 
     } else if (!!loggedUser.healthcarePartyId) {
       const hcpDataOwner = await authenticationResult.medTechApi.healthcareProfessionalApi.getHealthcareProfessional(loggedUser.healthcarePartyId);
-      if (!hcpDataOwner) throw new Error(`Healthcare Professional with id ${loggedUser.healthcarePartyId} does not exist`);
+      if (!hcpDataOwner) throw this.errorHandler.createErrorWithMessage(`Healthcare Professional with id ${loggedUser.healthcarePartyId} does not exist`);
```

Comment:
Impossible to find the healthcare party ${loggedUser.hcpId} apparently linked to the user ${loggedUser.id}. Are you sure this healthcarePartyId is correct ?